src/rules/declaration-empty-line-before/__tests__/index.js

```diff
@@ -191,6 +191,37 @@ testRule(rule, {
 
 testRule(rule, {
   ruleName,
+  config: [ "always", { ignore: ["after-declaration"] } ],
+
+  accept: [ {
+    code: "\ntop: 15px; top: 15px;\n",
+  }, {
+    code: "\r\ntop: 15px; top: 15px;\r\n",
+  }, {
+    code: "\ntop: 15px;\ntop: 15px;\n",
+  }, {
+    code: "\r\ntop: 15px;\r\ntop: 15px;\r\n",
+  }, {
+    code: "\ntop: 15px;\n\ntop: 15px;\n",
+  }, {
+    code: "\r\ntop: 15px;\r\n\r\ntop: 15px;\r\n",
+  } ],
+
+  reject: [ {
+    code: "a {\n top: 15px;\n}",
+    message: messages.expected,
+    line: 2,
+    column: 2,
+  }, {
+    code: "a {\r\n top: 15px;\r\n}",
+    message: messages.expected,
+    line: 2,
+    column: 2,
+  } ],
```

Comment:
Please add a couple of reject tests ensuring that declarations after *non-*declarations still trigger the relevant warning.